### PR TITLE
Remove breadcrumbs [Prev: uses bootstrap breadcrumbs in navigation]

### LINKED ISSUE #113: Rewrite Breadcrumbs
**Problem:** The present breadcrumbs are not properly visible. It is conflicting with other elements at view side.

**Image:**
problem_breadcrumbs

**Solution:**:
Use [Bootstrap Breadcrumb classes](http://getbootstrap.com/components/#breadcrumbs).

### PR DESCRIPTION
**Pull request by curioswati**
fixed #113 
In regional tab, the create and edit page lacks headers so the breadcrumbs might not be visible properly. Otherwise all are consistent.

### CODE REVIEW

**Comment by vnbang2003** [comment]
@curioswati Screenshot please ?

**Comment by curioswati** [comment]
wye_breadcrumbs_login
wye_breadcrumbs_regions

This one from regions/add location

wye_breadcrumbs_regions_location_add

**Comment by ultimatecoder** [comment]
@vnbang2003  @curioswati  Considering third image, I think there should be enough margin form the logo image. See, Logo is overriding Breadcrumbs.

**Comment by curioswati** [comment]
@ultimatecoder I think if we add headers to those pages then this problem will be solved. Are we not planning to add headers there?

File: wye/templates/base.html [hunk 1]
```diff
@@ -61,7 +61,9 @@
                 {% block sub_header %}{% endblock %}
 
                 {% block breadcrumbs %}
-                    <a href="{% url 'home-page' %}">Home</a>
+                    <ol class="breadcrumb">
```
Comment (ultimatecoder) [inline wye/templates/base.html]: @vnbang2003  This looks very ugly. I think we should decide other place for this.
home pythonexpress

Comment (theskumar) [inline wye/templates/base.html]: On homepage the `breadcrumb` can be removed.

**Comment by theskumar** [comment]
@curioswati See if you can comeup with a solution where breadcrumbs are properly aligned without depending on another element i.e. header in our case. There will many pages where we might not have a dedicated header.

@Praseetha-KR can you help!

**Comment by theskumar** [comment]
@curioswati can you change the hardcoded urls like `/workshop` to `{% url '<view_name>' %} pattern. It will definitely help us with fewer 404s.

**Comment by theskumar** [comment]
@curioswati have a look at https://django-bootstrap-breadcrumbs.readthedocs.org/en/latest/ it might help write breadcrumbs which are more readable.

**Comment by curioswati** [comment]
ok @theskumar . I will see that and make the appropriate changes.

**Comment by Praseetha-KR** [comment]
**Layout 1:** *breadcrumb placed below page header*
breadcrumbs_below_header

Breadcrumb is the secondary navigation to website and is usually placed below main navbar and above page header. So this is not a recommended layout when the information architecture is considered.

**Layout 2:** *breadcrumb placed above page header*
breadcrumbs_above_header

There is a design issue with this layout - you can see the links are repeating: 
   1. First level (Home link) and logo click both redirect to index page
   2. Second level (here 'Regions' link) is repeated in navbar
   3. Third level is the same as current page header

So it is a duplication of navigation system.

**Layout 3:** *without breadcrumb*
without_breadcrumbs

Remove breadcrumbs and highlight the current navbar item, this can be done only if we have 3 level hierarchy across the website.

@theskumar What is the best approach here?

**Comment by theskumar** [comment]
@Praseetha-KR 

- Layout 2: Definitely not. 
- Layout 1: Might work, you have point on duplicated stuff which I totally agree. Layout 1, will not be very mobile friendly as well.
- Layout 3: Is what i'm voting on. Let's keep things simple. Most of the pages have already been planned, I believe we don't any plans adding nested pages.

**Comment by curioswati** [comment]
so should I remove the breadcrumbs?

**Comment by saurabh-fueled** [comment]
@curioswati Go ahead and remove it, if we need it in future, we will create a ticket.

**Comment by theskumar** [comment]
@curioswati go head and remove it. We will create a new ticket, if/when we need it.

**Comment by Praseetha-KR** [comment]
Navbar current page highlight added in a6c38249efa2e0e6ba22788e008e94d31e9b432c

**Comment by vnbang2003** [comment]
@curioswati Can you update code with latest changes and send update pull request

**Comment by vnbang2003** [comment]
@curioswati Can you resolve conflicts and update PR Please.

**Comment by curioswati** [comment]
@vnbang2003 I didn't understand, what I have to do. I have removed the breadcrumbs and the changes are included in the PR. Do I need to create another PR?

**Comment by vnbang2003** [comment]
@curioswati I can't accept this pull request as i see there are few conflicts which needs to resolved.
Looks like you and @Praseetha-KR worked on same page and you have older version of file.
Two option (1) Resolve conflicts on your local by update code.
(2) Rebase code and apply changes on update files.

**Comment by curioswati** [comment]
ok. I am doing it.

**Comment by theskumar** [comment]
@curioswati i've resolved conflict and merged your changes at https://github.com/pythonindia/wye/commit/208344849b9c1b34a167a5efdf18e27512fa6c69. The required changes are not in master.

**Pull request closed without merging**